src/bulk_transfer.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import bittensor
+
+# Initialize the wallet and subtensor
+wallet = bittensor.wallet(name="miner")
+subtensor = bittensor.subtensor("ws://100.28.51.29:9945")
+
+# Read the CSV file and extract Coldkeys
+coldkeys = []
+with open("tTAO requests.csv", mode='r') as file:
+    csv_reader = csv.DictReader(file)
+    for row in csv_reader:
+        coldkeys.append(row["Coldkey"])
+
+# Read the funded coldkeys from funded_coldkeys.txt
+funded_coldkeys = []
+try:
+    with open("funded_coldkeys.txt", mode='r') as funded_file:
+        funded_coldkeys = [line.strip() for line in funded_file]
+except FileNotFoundError:
+    print("funded_coldkeys.txt not found. Proceeding with an empty list of funded coldkeys.")
+
+# Filter out already funded coldkeys
+coldkeys_to_fund = [coldkey for coldkey in coldkeys if coldkey not in funded_coldkeys]
+
+# Transfer 5 TAO to each Coldkey
+total_coldkeys = len(coldkeys)
+for index, coldkey in enumerate(coldkeys_to_fund):
+    try:
+        subtensor.transfer(wallet=wallet, dest=coldkey, amount=5.0)
```

Comment:
nitpick: weren't we gonna send 2 instead of 5 🤔 ? (Might be nothing, just asking to have consensus)

Reply:
during yesterday's standup @5u6r054 suggested 5 so that people are allowed to fail a couple of times before successfully registering their miner